packages/components/src/components/link-group/style.scss

```diff
@@ -8,25 +8,30 @@
 		padding: 0;
 	}
 
-	nav.horizontal ul {
+	ol.horizontal,
+	ul.horizontal {
```

Comment:
Martin hatte Recht, wir können hier `:is` verwenden:

`:is(ol, ul).horizontal`